tensorboard/plugins/histogram/summary_v2.py

```diff
@@ -471,13 +475,10 @@ def when_single_value():
                 # Left and right edges are the same for single value input.
                 edges = tf.fill([bucket_count], max_)
                 # Counts for the first {bucket_count - 1} buckets [v, v) are 0.
-                zero_bucket_counts = tf.constant(
-                    [0] * (bucket_count - 1), dtype=tf.int32
-                )
+                zero_bucket_counts = tf.repeat([0], bucket_count - 1)
```

Comment:
I know that `tf.repeat()` was my recommendation... but now that I'm looking at the code again, it's a bit odd that we're constructing two arrays here and using `tf.fill([length], value)` for one, and then `tf.repeat([value], length)` for the other - exactly opposite approaches!

Optional, but probably more consistent to do `tf.fill([bucket_count - 1], 0)` instead here.

Reply:
Done.